scripts/links_checker/tests/common/arguments/parser/links_checker_arguments_parser_test.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:args/args.dart';
+import 'package:links_checker/common/arguments/models/links_checker_arguments.dart';
+import 'package:links_checker/common/arguments/parser/links_checker_arguments_parser.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group("LinksCheckerArgumentsParser", () {
+    test(
+      ".configureArguments() configures the given arg parser to accept the paths option",
+      () {
+        final parser = ArgParser();
+        final argumentsParser = _ArgumentsParserFake();
+
+        argumentsParser.configureArguments(parser);
+
+        final options = parser.options;
+
+        expect(options, contains(LinksCheckerArgumentsParser.paths));
+      },
+    );
+  });
+}
```

Comment:
Should we add the following test case:
- `. parseArgResults() parses the given paths option value to a links checker arguments instance with a list of file paths`
?